kafka-elastic/src/main/java/com/baeldung/kafka/service/KafkaProducerService.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.kafka.service;
+
+import com.baeldung.kafka.model.NotificationModel;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
```

Comment:
We avoid using Lombok in articles that are not Lombok-specific

Reply:
@Service is a spring stereotype annotation and not a Lombok annotation

Reply:
I was talking about Slf4j

Reply:
Not resolved

Reply:
removed @Slf4j